src/Servers/IIS/IIS/src/Core/IISHttpContext.IO.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.AspNetCore.Server.IIS.Core;
 
 internal partial class IISHttpContext
 {
+    const ushort RESPONSE_BODY_MAX_CHUNKS = 65533;
```

Comment:
```suggestion
    const ushort responseMaxChunkBytes = 65533;
```

Reply:
According to the [documentation](https://docs.microsoft.com/en-us/iis/web-development-reference/native-code-api-reference/ihttpresponse-writeentitychunks-method), this is not the size in bytes of the chunks, but the maximum number of `HTTP_DATA_CHUNK` that it's possible to feed to the function. 

I will adapt the code to meet the const code convention, but I believe a better name is `responseMaxChunks`